09-schema/01-concepts.md

```diff
@@ -16,7 +16,7 @@ To try the following examples with one of the TypeDB clients, follows these [Cli
 
 <div class="note">
 [Important]
-Don't forget to `commit` after executing a `define` query. Otherwise, anything you have defined is NOT committed to the original database that is running on the TypeDB server.
+Don't forget to `commit` after executing a `define` query. Otherwise, anything we have defined is NOT committed to the original database that is running on the TypeDB server.
```

Comment:
```suggestion
Don't forget to `commit` after executing a `define` query. Otherwise, the resulting state of that query will NOT be committed to the database.
```